2023-risks/M8: Insufficient Cryptography

```diff
@@ -0,0 +1,106 @@
+---
+
+layout: col-sidebar
+title: "M8: Insufficient Cryptography"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Individuals who possess physical proximity to data that has been inadequately encrypted or mobile malware that operates on behalf of an adversary can compromise the security of the system.
+
+# Attack vectors
+
+**Exploitability Average**
+
+Any API call that is accessible to potential attackers can be exploited as an attack vector. An attack vector refers to a specific method or pathway through which an attacker can gain unauthorized access, manipulate data, or compromise the security of the mobile application.
+
+# Security weakness
+
+**Prevalence COMMON** <br /> 
+**Detectability Average**
+
+To exploit this vulnerability, an adversary must effectively decrypt encrypted code or sensitive data, exposing it in an unencrypted form. This vulnerability arises from weak encryption algorithms or flaws present within the encryption process itself.
```

Comment:
It also includes bad crypto configuration, like keys/certificates that are too short, right?